src/main/cms/cms_menu_imu.c

```diff
@@ -160,6 +160,8 @@ static OSD_Entry cmsx_menuPidEntries[] =
     { "YAW   D", OME_UINT8, NULL, &(OSD_UINT8_t){ &tempPid[PID_YAW][2],   0, 200, 1 }, 0 },
 
     { "BACK", OME_Back, NULL, NULL, 0 },
+    //You're welcome, mikeller.
+    { "SAVE&EXIT",   OME_OSD_Exit, cmsMenuExit,   (void *)CMS_EXIT_SAVE, 0},
```

Comment:
Why was this unrelated change not removed from this pull request as requested?

If you are not going to change the problems that are pointed out by reviewers then this is a waste of time and we should close this.